inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -655,11 +662,60 @@ class CompareNodesAttributes {
     attr_comparison::ReadAndCompareAttributes m_compare_attr;
 };
 
+class Comparator {
+public:
+    using CmpValues = FunctionsComparator::CmpValues;
+    using Result = FunctionsComparator::Result;
+    using ComparedNodes = std::pair<ngraph::Node*, ngraph::Node*>;
+
+    explicit Comparator(CmpValues f) : m_comparition_flags(f) {}
+
+    Result compare(
+        const std::shared_ptr<ngraph::Function>& f1, const std::shared_ptr<ngraph::Function>& f2);
+
+    Result compare(ngraph::Node* node1, ngraph::Node* node2) {
+        std::stringstream errors;
```

Comment:
Passing `stringstream` to execution is required because for now we have less important errors gather in `stringstream` called `err_log`. If this policy (minor and major error) will remove src will be simpler.